src/App.js

```diff
@@ -14,7 +14,8 @@ import ErrorPage from './views/ErrorPage';
 import { useAppSettings } from './store/store.js';
 import { PrintContextProvider } from './store/printContext';
 import { useAppInitializer } from './hooks';
-import Layout from './views/Layout';
+//import Layout from './views/Layout';
```

Comment:
Please remove commented line

Reply:
whoops, forgot to add that to the commit, my bad